docs/_build/html/_sources/index.rst.txt

```diff
@@ -0,0 +1,20 @@
+.. Snowplow Python Tracker documentation master file, created by
+   sphinx-quickstart on Wed Jul 20 14:00:53 2022.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to Snowplow Python Tracker's documentation!
+===================================================
+
```

Comment:
Not that important but could we perhaps paste some short intro what is the tracker here? Maybe just copy the overview from the README, it looks a bit empty. Similar to [the front page of the C++ tracker API docs](https://snowplow.github.io/snowplow-cpp-tracker/).